src/components/ec-table-footer/ec-table-footer.vue

```diff
@@ -23,25 +23,23 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/tools/typography';
-@import '../../scss/settings/colors/index';
+<style>
 
 .ec-table-footer {
-  @include small-text;
+  @apply tw-small-text;
 
   &__cell {
-    padding: 0 16px;
-    width: 100%;
+    @apply tw-py-0 tw-px-16;
+    @apply tw-w-full;
+    @apply tw-bottom-0;
+    @apply tw-bg-gray-7;
+
     border-radius: 0 0 5px 5px;
```

Comment:
Here you can use this class tw-rounded-b

Reply:
Done thanks!